### ISSUE TITLE
Can't  sudo root in Tabby terminal

### ISSUE THREAD

**Issue by MatchaMusturd** [Eugeny/tabby#10048]
Version: 1.0.215
Platform: Linux arm64 6.6.30-02726-gb84bbdf6955e
Plugins: fit-screen, background
Frontend: xterm-webgl

sudo: The "no new privileges" flag is set, which prevents sudo from running as root.
sudo: If sudo is running in a container, you may need to adjust the container configuration to disable the flag.

**Comment by q3aql** [comment]
I fixed the problem editing "/usr/share/applications/tabby.desktop" and adding the flag "--no-setuid-sandbox".

My .desktop file:

```
[Desktop Entry]
Name=Tabby
Exec=/opt/Tabby/tabby --no-sandbox --no-setuid-sandbox %U
Terminal=false
Type=Application
Icon=terminal
StartupWMClass=tabby
Comment=A terminal for a modern age
Categories=Utility;TerminalEmulator;System;
```

Try it

**Comment by OpenCode** [comment]
> I fixed the problem editing `/usr/share/applications/tabby.desktop` and adding the flag `--no-setuid-sandbox`.
> 
> My .desktop file:
> 
> ```
> [Desktop Entry]
> Name=Tabby
> Exec=/opt/Tabby/tabby --no-sandbox --no-setuid-sandbox %U
> Terminal=false
> Type=Application
> Icon=terminal
> StartupWMClass=tabby
> Comment=A terminal for a modern age
> Categories=Utility;TerminalEmulator;System;
> ```
> 
> Try it

Thanks. It works!

**Comment by Shad0w1998** [comment]
Worked for me! Thanks a lot!

**Comment by Readus-01** [comment]
Worked for me too! 
perfect , Thanks a lot!

**Comment by pattisonmichael** [comment]
this worked for a while but since a few days the issue is back and i have the same config as mentioned above.
It works for the very first connection and then any other connection just throws this error till i restart Tabby.

**Comment by Nirzak** [comment]
Yeah same here. I issue comes back after some time and then never fix until I fully restart tabby closing all open windows.

**Comment by DenZen1988** [comment]
Same issue, but only on version 1.0.230 (ZorinOS/Ubuntu). As a workaround I downgraded to version 1.0.229.
